### ISSUE TITLE
intl problem

### ISSUE THREAD

**Issue by cnkygmr** [sbis04/video_trimmer#232]
```
Because every version of video_trimmer from path depends on intl ^0.18.1 and xxx depends on intl ^0.19.0, video_trimmer from path is forbidden.
So, because xxx depends on video_trimmer from path, version solving failed.
```

**Comment by UtkuDalmaz** [comment]
Please update the package to support intl ^0.19.0 @sbis04

**Comment by Shvet** [comment]
You can temperory override dependecy till new update does not pushed to pub.

**Comment by ChristineWasike** [comment]
@Shvet how does one go about this exactly?

**Comment by YofarDev** [comment]
in your pubspec.yaml you can add :

# temporary (until video_trimmer update)
dependency_overrides:
  intl: 0.19.0

**Comment by sbis04** [comment]
Should be fixed in the latest version `4.0.1`!
